src/main/java/org/mockito/internal/session/DefaultMockitoSessionBuilder.java

```diff
@@ -0,0 +1,33 @@
+package org.mockito.internal.session;
+
+import org.mockito.MockitoSession;
+import org.mockito.internal.framework.DefaultMockitoSession;
+import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.quality.Strictness;
+import org.mockito.session.MockitoSessionBuilder;
+
+public class DefaultMockitoSessionBuilder implements MockitoSessionBuilder {
+
+    private Object testClassInstance;
+    private Strictness strictness;
+
+    @Override
+    public MockitoSessionBuilder initMocks(Object testClassInstance) {
```

Comment:
I think the new name will speak more than the old name.
On MockitoAnnotations, this `initMocks` tickled me for quite some time, but since I use moregularly the JUnit runner or the JUnit rule, I didn't acted on it.

I think we could deprecate the `MockitoAnnotations.initMocks` and add the alternative method, it could in another PR though.